app/Migrations/Version20240830071940.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/*
+ * Adding primary keys
+ * */
+class Version20240830071940 extends AbstractMigration
```

Comment:
I would advise to change the date, so it comes after the last migration to make sure it gets run.